qemu/tests/slof_memory.py

```diff
@@ -0,0 +1,58 @@
+"""
+slof_memory.py include following case:
+ 1. CAS(client-architecture-support) response with large maxmem.
+"""
+import logging
+
+from virttest import error_context
+from provider import slof
+from virttest import env_process
+from virttest import utils_net
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    Verify SLOF info with maxmem options.
+
+    Step:
+     1. Boot a guest with "maxmem=512G".
+      a. check no errors from output of SLOF.
+      b. log in guest successfully.
+      c. ping external host ip successfully.
+     2. Shutdown the guest then boot it again with "maxmem=1024G".
+      a. check no errors from output of SLOF.
+      b. log in guest successfully.
+      c. ping external host ip successfully.
+
+    :param test: Qemu test object.
+    :param params: Dictionary with the test parameters.
+    :param env: Dictionary with test environment.
+    """
+    start_pos = 0
+    for mem in params['maxmem_mem_list'].split():
+        params['maxmem_mem'] = mem
+        params['start_vm'] = 'yes'
```

Comment:
I think you can put `start_vm = yes` in the cfg

Reply:
Thanks for your comments,   have updated.